pkg/util/informermanager/multi-cluster-manager.go

```diff
@@ -34,24 +46,29 @@ type MultiClusterInformerManager interface {
 	// IsManagerExist checks if the informer manager for the cluster already created.
 	IsManagerExist(cluster string) bool
 
-	// Start will run all informers for a specific cluster, it accepts a stop channel, the informers will keep running until channel closed.
+	// Start will run all informers for a specific cluster.
 	// Should call after 'ForCluster', otherwise no-ops.
-	Start(cluster string, stopCh <-chan struct{})
+	Start(cluster string)
+
+	// StopAndDeleteCluster will stop all informers for a specific cluster, and delete the cluster from informer managers.
+	StopAndDeleteCluster(cluster string)
```

Comment:
```suggestion
	Stop(cluster string)
```
Why not keep it simple?

Reply:
Because it would not only stop the cluster but also delete cluster from the informemanager map. `StopAndDeleteCluster` may be more distinct.